samples/Tests/SearchResults.json

```diff
@@ -6,18 +6,18 @@
 		{
 			"type": "TextBlock",
 			"text": "3 results for 3d printer filament",
-            "bold": true,
-            "wrap": true,
-            "size": "large",
-            "horizontalAlignment": "center"
+			"bold": true,
+			"wrap": true,
+			"size": "large",
+			"horizontalAlignment": "center"
         },
 		{
 			"type": "Input.ChoiceSet",
 			"id": "filament",
 			"style": "expanded",
 			"isMultiSelect": false,
 			"value": "1",
-            "wrap": true,
+			"wrap": true,
```

Comment:
>"wrap": true, [](start = 3, length = 13)

I'm looking at the test app results, and it doesn't look like these choices are wrapping...